docs/reference/Func/SECValidateAccessToken.md

```diff
@@ -33,9 +33,9 @@ Input :
 
 dwFlags  -  Modify behavior from defaults. See fJWT_validate_* below.
 
-*vpOptionalParams  -  pszCustomClaimName.
+*vpOptionalParams  -  pszCustomClaimName value can be filled with using this structure. Please refer this [JWT_VALIDATE_OPTIONAL_PARAMS.md](/domino-c-api-docs/reference/Data/JWT_VALIDATE_OPTIONAL_PARAMS.md).
 
-dwMaxEmailSize  -  ResourceCallback.
+dwMaxEmailSize  -  MaxEmailSize.
```

Comment:
dwMaxEmailSize  -  Maximum length of email buffer size.

Reply:
Modified